### PR TITLE
Added  Feature to stop streaming in remove() method(#4085)(#4140).

### PR DESCRIPTION
**Pull request by singhvisha**
Resolves (#4085 )(#4140 )

 Changes: 
Added feature in the remove() method (in p5.Element) to stop streaming caused the createCapture().

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated
- [x] [Benchmarks] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/master/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/master/contributor_docs#unit-tests
[Benchmarks]: https://github.com/processing/p5.js/blob/master/contributor_docs/benchmarking_p5.md

### CODE REVIEW

**Comment by singhvisha** [comment]
@limzykenneth @stalgiag 
Hi,Could you please review the above changes?

File: src/dom/dom.js [hunk 1]
```diff
@@ -2063,6 +2063,19 @@ p5.Element.prototype.size = function(w, h) {
  * </code></div>
  */
 p5.Element.prototype.remove = function() {
+  // stops all audios/videos and detach all devices like microphone/camera
+  // used as input/output for audio/video.
+  if (this instanceof p5.MediaElement) {
+    var stream = this.elt.srcObject;
+    var tracks = stream.getTracks();
```
Comment (lmccart) [inline src/dom/dom.js]: can you change these to `const`?

Comment (lmccart) [reply src/dom/dom.js]: this could be combined into one line to condense:
`let tracks = this.elt.srcObject.getTracks();`

**Comment by singhvisha** [comment]
hi @lmccart 
I  modified the code as you have suggested.Could you please review it?

**Comment by lmccart** [comment]
@all-contributors please add @singhvisha for code

**Comment by lmccart** [comment]
looks great, thanks!

**Pull request merged**